src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -1,11 +1,29 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <Import Project="..\..\PowerShell.Common.props" />
   <PropertyGroup>
     <Description>PowerShell's System.Management.Automation project</Description>
     <NoWarn>$(NoWarn);CS1570;CS1734;CA1416</NoWarn>
     <AssemblyName>System.Management.Automation</AssemblyName>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- we persist source generator files under 'gen' folder so that they are visible to IDEs -->
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>gen\SourceGenerated</CompilerGeneratedFilesOutputPath>
+  </PropertyGroup>
```

Comment:
Generate the file, so VSCode and Visual Studio can always resolve the generated members.
Those files will be removed when building again, by `RemoveDir` in `PowerShell.Common.props`.

Reply:
I pushed a commit a day before (but you reverted now) and removed this old workaround because all works well in VS Code. We have no need to emit generated files.
Perhaps you need re-configure your local environment.
I use VS Code 1.72 and C# extension 1.25
![image](https://user-images.githubusercontent.com/22290914/194461998-943a41c5-b97a-4cc2-9e98-148fd160bbc3.png)

Reply:
You didn't see my reply to your comment? It's here: https://github.com/PowerShell/PowerShell/pull/15603#discussion_r989291958, and quoted below:

> It doesn't work in the following scenarios, for both VSCode and Visual Studio, which are quite common in development:
>1. Run `Start-PSBuild` first, then open `System.Management.Automation` in VSCode
> 2. Open `System.Management.Automation` in VSCode to use an existing build, like from yesterday.

Reply:
Sorry, too many comments already....